xain_fl/fl/coordinator/controller.py

```diff
@@ -123,4 +123,5 @@ def select_ids(self, participant_ids: List[str]) -> List[str]:
 
         num_ids_to_select = self.get_num_ids_to_select(len(participant_ids))
         ids = np.random.choice(participant_ids, size=num_ids_to_select, replace=False)
-        return ids.tolist()
+        list_ids: List[str] = ids.tolist()
```

Comment:
it's a little annoying that mypy isn't smart enough to infer this!

Reply:
unfortunately there is no way to turn this off and have other benefits of mypy